src/main/java/ch/njol/skript/effects/EffInvulnerability.java

```diff
@@ -40,8 +40,8 @@ public class EffInvulnerability extends Effect {
 	
 	static {
 		Skript.registerEffect(EffInvulnerability.class,
-			"make %entities% invulnerable",
-			"make %entities% (not invulnerable|vulnerable)");
+			"make %entities% (invulnerable|invincible)",
+			"make %entities% (not (invulnerable|invincible)|vulnerable|vincible)");
```

Comment:
At this point couldn't we just have a flag for [in]?

Reply:
Sure, but I left it like this because I think it's easier to read for users.

Reply:
Also it's helpful for docs searching instead of using `@Keywords`